### ISSUE TITLE
HITEMP/HITRAN quantum states filtering

### ISSUE THREAD

**Issue by HajimeKawahara** [HajimeKawahara/exojax#337]
Quantum states filtering was implemented for ExoMol #336 but not yet HITEMP and HITRAN.
It looks the original file contains such information:

 - "global_upper_quanta": "Electronic and vibrational quantum numbers and 
 -  "global_lower_quanta": "Electronic and vibrational quantum numbers and labels for the lower state of a transition"

**Comment by HajimeKawahara** [comment]
@YuiKasagi

**Comment by erwanp** [comment]
Its implemented in Radis for CO2 and CO (therefore in the api)

**Comment by erwanp** [comment]
Parsing was also done for many hitran molecules  https://github.com/radis/radis/pull/441

**Comment by HajimeKawahara** [comment]
Oh, cool! I'll try it.

**Comment by erwanp** [comment]
Just check that it's what you need in Exojax. We can discuss on Slack
